docs/DafnyRef/Attributes.md

```diff
@@ -344,136 +326,220 @@ recursion was explicitly requested.
+### 22.1.13. `{:timeLimitMultiplier X}`
 This attribute may be placed on a method or function declaration
 and has an integer argument. If `{:timeLimitMultiplier X}` was
 specified a `{:timelimit Y}` attributed is passed on to Boogie
 where `Y` is `X` times either the default verification time limit
 for a function or method, or times the value specified by the
 Boogie `timelimit` command-line option.
 
-### 22.1.16. trigger
+### 22.1.14. `{:trigger}` {#sec-trigger}
 Trigger attributes are used on quantifiers and comprehensions.
 They are translated into Boogie triggers.
 
-### 22.1.17. typeQuantifier
+### 22.1.15. `{:typeQuantifier}`
 The `{:typeQuantifier}` attribute must be used on a quantifier if it
 quantifies over types.
 
 
-## 22.2. Boogie Attributes
-Use the Boogie "/attrHelp" option to get the list of attributes
-that Boogie recognizes and their meaning. Here is the output at
-the time of this writing. Dafny passes attributes that have
-been specified to Boogie.
+## 22.2. Boogie-specific verification attributes
+Dafny passes verification attributes that have been specified to its dependency Boogie.
+Use the `/attrHelp` option to get the list of attributes
+recognized by Boogie and their meaning.
+
+### 22.2.1. Verification attributes on top-level declarations
+
+#### 22.2.1.1. `{:ignore}`
+Ignore the declaration (after checking for duplicate names).
+
+#### 22.2.1.2. `{:extern}` {#sec-extern}
+
+If a ``ClassDecl``, a ``MethodDecl`` or a ``FunctionDecl`` has an `{:extern}` attribute,
+then:
+* The compiler is going to emit nothing for it, it assumes it exists in the linked files
+* Methods and functions can have [`requires` clauses](#sec-requires-clause) and [`ensures` clauses](#sec-ensures-clause).
+* Methods should not have any body, since they are unchecked and opaque.
+* Functions can have a body which is used only for verification purposes.
+
+The difference with [`{:axiom}`](#sec-axiom) is that the compiler will still emit code for an [`{:axiom}`](#sec-axiom), if it is a [`function method`, a `method` or a `function by method`](#sec-function-declarations) with a body.
+
+#### 22.2.1.3. `{:checksum <string>}`
+
+Attach a checksum to be used for verification result caching.
+
+### 22.2.2. Verification attributes on functions and methods
+
+#### 22.2.2.1. `{:inline N}`
+     
+Inline a given procedure (can be also used on implementation).
+N should be a non-negative number and represents the inlining depth.
+With `/inline:assume`, call is replaced with `assume false` once inlining depth is reached.
+With `/inline:assert`, call is replaced with `assert false` once inlining depth is reached.
+With `/inline:spec`, call is left as is once inlining depth is reached.
+With the above three options, methods with the attribute `{:inline N}` are not verified.
+With `/inline:none` the entire attribute is ignored.
+
+#### 22.2.2.2. `{:verify false}` {#sec-verify}
+     
+Skip verification of an implementation altogether
```

Comment:
```suggestion
Skip verification of an implementation altogether.
```

Also what does "implementation" mean here? Won't it also skip the well-formedness proof obligations?

Reply:
Removed reference to Boogie, and described in better detail.